src/scss/mofo-bootstrap.scss

```diff
@@ -9,18 +10,17 @@
 @import 'custom/colors';
 
 // Core variables and mixins
+@import '../../node_modules/bootstrap/scss/functions';
 @import '../../node_modules/bootstrap/scss/variables';
 @import './overrides/variables';
 @import '../../node_modules/bootstrap/scss/mixins';
-@import './overrides/mixins';
 
 // Reset and dependencies
-@import '../../node_modules/bootstrap/scss/normalize';
+@import '../../node_modules/bootstrap/scss/root';
+@import '../../node_modules/bootstrap/scss/reboot';
```

Comment:
Moving this up has the potential to significantly change the way things work, is there a reason it moved?

Reply:
Is according to actual bootstrap.scss from v4.1.3